### PR TITLE
Create file

### PR DESCRIPTION
**Pull request by hanegbi**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                src_list.append(src)
+                print(str(photo_id) + '\t' + src)
+                photo_id += 1
+            finally:
+                next_photo = self.driver.find_element(By.XPATH, self.NEXT_ELEMENT)
+                next_photo.click()
+        return src_list
+
+    def download_photo(self, src, photo_id):
+        urllib.request.urlretrieve(src, str(self.get_count_photos() - photo_id + 1) + '_facebook_photos_of_me.jpg')
```
Comment (prussyuval) [inline main.py:71]:
```suggestion
        urllib.request.urlretrieve(src, f'{self.get_count_photos() - photo_id + 1}_facebook_photos_of_me.jpg')
```

File: main.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        email = self.driver.find_element(By.NAME, 'email')
+        email.clear()
+        email.send_keys(self.EMAIL)
+        password = self.driver.find_element(By.NAME, 'pass')
+        password.clear()
+        password.send_keys(self.PASSWORD)
+        password.send_keys(Keys.RETURN)
+
+    def get_username(self):
+        username_class = self.driver.find_element(By.XPATH, "//*[@id='u_0_a']/div[1]/div[1]/div/a")
```
[the remark this answers is not in the document]
Comment (hanegbi) [reply main.py:41]: Yes

File: main.py (hunk 1 above)
Comment (hanegbi) [reply main.py:71]: Nice it's better

File: main.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    def download_photo(self, src, photo_id):
+        urllib.request.urlretrieve(src, str(self.get_count_photos() - photo_id + 1) + '_facebook_photos_of_me.jpg')
+        print(str(photo_id) + " of " + str(self.get_count_photos()) + " photos downloaded")
+
+    def download_all_photos(self, src_list):
+        for photo_id, src in enumerate(src_list, start=1):
+            threading.Thread(target=self.download_photo, args=(src, photo_id,)).start()
+
+    @staticmethod
+    def get_count_photos():
```
[the remark this answers is not in the document]
Comment (hanegbi) [reply main.py:79]: LIT

File: main.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        return count_photos
+
+    def download_photos_of_me(self):
+        self.log_in()
+        time.sleep(3)
```
[the remark this answers is not in the document]
Comment (hanegbi) [reply main.py:84]: Co0L

File: main.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        self.driver.get(self.URL + self.get_username() + '/photos')
+        self.scroll_down()
+        global count_photos
+        count_photos = len(self.driver.find_elements(By.CLASS_NAME, 'uiMediaThumbImg'))
+        print(count_photos)
+        self.driver.find_element_by_class_name('uiMediaThumbImg').click()
+        src_list = []
+        photo_id = 1
```
[the remark this answers is not in the document]
Comment (hanegbi) [reply main.py:92]: Starting from 1
Maybe setcreate_photos_src_list(photo_id=1, src_list)?

File: main.py [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    def download_photos_of_me(self):
+        self.log_in()
+        time.sleep(3)
+        self.driver.get(self.URL + self.get_username() + '/photos')
+        self.scroll_down()
+        global count_photos
+        count_photos = len(self.driver.find_elements(By.CLASS_NAME, 'uiMediaThumbImg'))
+        print(count_photos)
+        self.driver.find_element_by_class_name('uiMediaThumbImg').click()
+        src_list = []
```
[the remark this answers is not in the document]
Comment (hanegbi) [reply main.py:91]: Just initializing the list

File: main.py [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    @staticmethod
+    def get_count_photos():
+        return count_photos
+
+    def download_photos_of_me(self):
+        self.log_in()
+        time.sleep(3)
+        self.driver.get(self.URL + self.get_username() + '/photos')
+        self.scroll_down()
+        global count_photos
```
[the remark this answers is not in the document]
Comment (hanegbi) [reply main.py:87]: OK

**Pull request closed without merging**